provisioning/dev/prometheus/prometheus.yml

```diff
@@ -0,0 +1,19 @@
+global:
+  scrape_interval: 10s
+scrape_configs:
+ - job_name: prometheus
+   static_configs:
+    - targets:
+       - 'prometheus:9090'
+ - job_name: templatesAPI
+   static_configs:
+    - targets:
+       - 'dev:9000'
+ - job_name: streamAPI
+   static_configs:
+    - targets:
+       - 'dev:9001'
+ - job_name: AppsProxy
```

Comment:
why not 'appsProxy' ?

Reply:
It's just name of job in prometheus, can be anything, not very important but I will change it

Reply:
aa81117023dea3abdf4eb9fdc29412d28460c402